kyo-combinators/shared/src/main/scala/kyo/Combinators.scala

```diff
@@ -227,8 +229,11 @@ extension [A, S](effect: A < S)
       * @return
       *   A computation that produces the result of this computation with Abort[Throwable] effect
       */
-    def explicitThrowable(using Flat[A], Frame): A < (S & Abort[Throwable]) =
-        Abort.catching[Throwable](effect)
+    def resurrect(using Flat[A], Frame): A < (S & Abort[Throwable]) =
+        Abort.run[Throwable](effect: A < (S & Abort[Throwable])).map:
```

Comment:
Is the explicit type annotation for the computation isn't necessary?

Reply:
Apparently not -- removed